src/Core/Domain/Employee/Command/AddEmployeeCommand.php

```diff
@@ -101,7 +107,8 @@ public function __construct(
         $languageId,
         $active,
         $profileId,
-        array $shopAssociation
+        array $shopAssociation,
+        bool $hasEnabledGravatar
```

Comment:
FInally we reach the moment where we want to update a Command 😄 . And in theory this is a BC break.

@eternoendless finally we have a real world usecase for this issue we knew would come

So ... nullable new argument or new Command ?